templates/taxi/car_detail.html

```diff
@@ -1,18 +1,26 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>
+  <h1 class="text-left">
     {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
+    
+      <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger ml-5">Delete</a>
+      <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary">Update</a>
+    
+  </h1>
+  <p>Manufacturer: {{ car.manufacturer.name }} ({{ car.manufacturer.country }})</p>
+  <h1>
+    <span>Drivers</span>
+    {% if user not in car.drivers.all %}
+      <a class="btn btn-primary ml-5" href="{% url 'taxi:driver-assign-to-car' user.id car.id %}">
```

Comment:
URL Tag Syntax: Ensure that the arguments in the `{% url %}` tag are separated by commas. It should be `{% url 'taxi:driver-assign-to-car' user.id, car.id %}`.